analysis/approval_status_testing.ipynb

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
I can't comment on the Precision oncolody coding systems file, but a README would be helpful for explaining the different sheets and their meaning

Reply:
I don't really know anything about that file -- it's been hanging out in the root for a while (I think before we started working here) so I just moved into /analysis for the sake of cleanliness

Reply:
Ope my b